src/v/redpanda/application.cc

```diff
@@ -1968,22 +1967,18 @@ void application::wire_up_bootstrap_services() {
                 = config::shard_local_cfg().rpc_server_tcp_recv_buf;
               c.tcp_send_buf
                 = config::shard_local_cfg().rpc_server_tcp_send_buf;
-              auto rpc_builder = config::node()
-                                   .rpc_server_tls()
-                                   .get_credentials_builder()
-                                   .get0();
               auto credentials
-                = rpc_builder
-                    ? rpc_builder
-                        ->build_reloadable_server_credentials(
-                          [this](
-                            const std::unordered_set<ss::sstring>& updated,
-                            const std::exception_ptr& eptr) {
-                              rpc::log_certificate_reload_event(
-                                _log, "Internal RPC TLS", updated, eptr);
-                          })
-                        .get0()
-                    : nullptr;
+                = net::build_reloadable_server_credentials_with_probe(
+                    config::node().rpc_server_tls(),
+                    "rpc",
+                    "",
+                    [this](
+                      const std::unordered_set<ss::sstring>& updated,
+                      const std::exception_ptr& eptr) {
+                        rpc::log_certificate_reload_event(
+                          _log, "Internal RPC TLS", updated, eptr);
+                    })
+                    .get0();
```

Comment:
nbitpick: `get0` -> `get()`

Reply:
many such cases

Reply:
> many such cases

It's probably not worth fixing them all.

Reply:
Yeah, just made sure there aren't any in this particular diff